src/services/emailService.js

```diff
@@ -0,0 +1,50 @@
+const nodemailer = require('nodemailer');
+const bcrypt = require('bcryptjs');
+const User = require('../models/User');
+const jwt = require('jsonwebtoken');
+const { secret } = require('../utils/parseToken');
+const { generatePassword } = require('../utils/generatePassword');
+
+async function resetPassword(requestBody) {
+    // TODO: Try to update the function later, with expired password or magic link..
+    const existingUser = await User.findOne({ email: requestBody.to });
+
+    if (!existingUser) {
+        throw new Error('User-Email not found!');
+    }
+
+    const newPassword =  generatePassword();
```

Comment:
extra space before calling "generatePassword()", are you using the same styling plugins as your teammates ?